test/functional/wallet-hd.py

```diff
@@ -83,7 +83,7 @@ def run_test (self):
         for _ in range(num_hd_adds):
             hd_add_2 = self.nodes[1].getnewaddress()
             hd_info_2 = self.nodes[1].validateaddress(hd_add_2)
-            assert_equal(hd_info_2["hdkeypath"], "m/0'/0'/"+str(_+1)+"'")
+            assert_equal(hd_info_2["hdkeypath"], "m/0'/0'/"+str(_)+"'")
```

Comment:
nit: might want to rename the variable to something other than underscore, which is only used to denote something that is never used.